Formula/usbutils.rb

```diff
@@ -0,0 +1,44 @@
+class Usbutils < Formula
+  desc "Provides 'lsusb' tool for getting detailed info about USB devices"
+  # Homepage for multiple Linux USB tools, 'usbutils' is one of them.
+  homepage "http://www.linux-usb.org/"
+  # The latest version we can use is 007 (from 2013), because the later
+  # versions depend on 'libudev': part of Systemd and not in Homebrew.
```

Comment:
We actually have this formula in homebrew.

Reply:
I'll edit this comment.
The real issue is that macOS doesn't expose devices in `/dev` like it's in linux.
I think Homebrew `systemd` is not sufficient for this to work.

@tormodvolden, the maintainer of dfu-util, says it's possible to upstream patch to be able to build usbutils without `libudev`.

Reply:
Maybe we can make the formula Linux only at first then? And try and patch it upstream to avoid keeping an old version forever.

Reply:
Nope, the whole point is to have real lsusb on macOS.

I think we can freeze at 007 on macOS and build latest tag on linux.

But I think it's more reasonable to make this formula macOS-only, as linux users usually have usbutils as part of installation.

Reply:
That's really at odds with https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff though. We don't want to have old versions of software if we can avoid it.

Reply:
It's still a way better to have a real `lsusb` than some wrapper shell script.

@tormodvolden has patch for compiling latest `usbutils` without `libudev`.
If not having the most recent version is the issue, I think, we can add the patch to this formula on macOS